CONTRIBUTING.md

```diff
@@ -95,30 +98,91 @@ we would appreciate if your pull request applies to the following points:
-  a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
-* We prefer small changes and short living branches. Please do not combine independent things in one pull request.
+  As such, all parties involved in a contribution must have valid ECAs. Additionally, commits can 
+  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
+* We prefer small changes and short living branches. Please do not combine independent things in one
+  pull request.
 
 
 * Excessive branching and merging can make git history confusing. With that in mind please
 
-    * [Squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing)
+    * [squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing)
       down to a few commits, or one commit, before submitting a pull request and
-    * [Rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your pull request changes on top of the current
-      master. Pull requests shouldn't include merge commits.
+    * [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your pull request changes on 
+      top of the current master. Pull requests shouldn't include merge commits.
 
 * Add meaningful tests to verify your submission acts as expected.
 
-* Where code is not self-explanatory add documentation providing extra clarification.
+* Where code is not self-explanatory, add documentation providing extra clarification.
 
-* Submit a draft pull request at early-stage and add people previously working on the same code as reviewer.
-  Make sure automatically checks pass before marking it as "ready for review":
+* Submit a draft pull request at early-stage and add people previously working on the same code as 
+  reviewer. Make sure automatically checks pass before marking it as "ready for review":
 
-    * _Intellectual Property Validation_ verifying the [Eclipse CLA](#eclipse-contributor-agreement) has been signed as
-      well as commits have been signed-off and
+    * _Intellectual Property Validation_ verifying the [Eclipse CLA](#eclipse-contributor-agreement) 
+      has been signed as well as commits have been signed-off and
     * _Continuous Integration_ performing various test conventions.
 
+## Project and Milestone Planning
+
+Milestones are planned for about 6 to 8 weeks. Future milestones describe an outlook on features to 
+be considered. The transparent process on GitHub enables participation. To decide what features and
+issues will be targeted, user requirements and sponsors are considered. Thus, everyone is able to
+influence the addressed developments and its priorities! 
+
+### Projects
+
+The [Github Projects page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/projects)
+provides a general overview of the project's working items. Every new issue needs to be assigned to
```

Comment:
I mean "every new". "Every accepted" will be put to the "backlog" column (referring to the comment above). This way, it is sufficient to take a look at the projects tab without missing an issue. And by doing so, one can easily use this as a starting of discussions to talk about what issues may be relevant for the backlog, what may be converted into a discussion and so on.  I can re-formulate this by using your words - I think they describe the difference very well and may also resolve your comment above.

Unless we say that only issues will be created, which we will definitely work on at some point. However, to be honest, we can never say that 100%, because any GitHub user can create an issue - whether it is a working item or not. By having two columns, we can easily see issues as e.g. "out of scope" or "need to be discussed first", leave them in the "open" column and check later again if this should be added to the backlog - whether it is addressed in the current or any further milestone.

Reply:
is that information ("backlog" vs. "open") also visible in the Issue itself? Would be great to have the board (or maybe multiple boards in future) just as a thing to increase the overview while still being optional.

Reply:
Yes, this information is visible: 
![image](https://user-images.githubusercontent.com/72392527/154252640-3d2343e3-3141-4bac-ab2e-2f354c4021c2.png)

Maybe we can say that there are four columns: "backlog", "in progress" etc.? Open issues are not automatically added to a project (this must be done manually anyway). And as soon as an issue is officially "a task to be definitely done", it is put to either the general project's board or a catena-x board or similar?

Reply:
@MoritzKeppler I've shortened the project section. Maybe you can take another look? I don't want to just resolve this comment and the one above, maybe others want to take a look at both of the proposals?